Libraries/LibGUI/GFilePicker.cpp

```diff
@@ -189,3 +231,17 @@ void GFilePicker::clear_preview()
     m_preview_name_label->set_text(String::empty());
     m_preview_geometry_label->set_text(String::empty());
 }
+
+bool GFilePicker::file_exists(const StringView& path)
+{
+    struct stat st;
+    int rc = stat(path.characters_without_null_termination(), &st);
```

Comment:
It's not safe to pass `StringView::characters_without_null_termination()` to a function that expects a null-terminated string. The whole reason it's so awkwardly named is to make you think about what you're doing :)
Unfortunately there's currently no way to pass the path length to `stat()`, so you'll have to construct a temporary `String`:
```suggestion
    int rc = stat(String(path).characters(), &st);
```